modules/consensus/database.go

```diff
@@ -78,6 +78,29 @@ var (
 	SiafundPool = []byte("SiafundPool")
 )
 
+type (
+	// dbBucket represents a collection of key/value pairs inside the database.
+	dbBucket interface {
+		Get(key []byte) []byte
+	}
+
+	// dbTx represents a read-only transaction on the database that can be used
+	// for retrieving values.
+	dbTx interface {
+		Bucket(name []byte) dbBucket
+	}
+
+	// boltTxWrapper wraps a bolt.Tx so that it matches the dbBucket interface.
```

Comment:
I expanded the comment to better explain in a6ca0ba035d8e3bfd12bd533a0f4f37d461835d4 (also `dbBucket` should have been `dbTx`). Let me know if that clarifies.